build.gradle

```diff
@@ -39,7 +39,7 @@ sourceSets {
 
 // Remove all .proto definition from the final build
 processResources {
-    exclude('**/*')
+    exclude('POGOProtos/')
```

Comment:
It won't work without the clean exclusion of the Proto files since the resources would also match the Regex and therefore be excluded on build. Thats why I included it here ;)